src/pres/loader/PGNLoader.js

```diff
@@ -209,6 +208,43 @@ export default class PGNLoader extends React.Component {
         this.props.variantChange(newVariant)
         trackEvent(Constants.EVENT_CATEGORY_PGN_LOADER, "VariantChange", newVariant)
     }
+    getInitiallyExpandedPanel(selectedSite, playerName) {
+        if (!selectedSite) {
+            return 'source'
+        } else if (!playerName) {
+            return 'user'
+        } else {
+            return 'filters'
+        }
+    }
+    getTimeControls() {
```

Comment:
Not sure if this name should have "initial" / "initially" in it or not (same with `getInitiallyExpandedPanel` above)